src/Plugins/ComplexCore/test/QuickSurfaceMeshFilterTest.cpp

```diff
@@ -15,7 +15,8 @@
 using namespace complex;
 using namespace complex::UnitTest;
 using namespace complex::Constants;
-
+namespace SurfaceMeshingTest
```

Comment:
```suggestion
namespace
```
Should be anonymous namespace.